File: scrape-data/src/Domain/Product.java
package Domain;

public class Product {
	Long productId;
	String productName;
	double price;
	String ImageUrl;
	String brand;
	String link;
	String website;
	public Product(){}
	

	public Long getProductId() {
		return productId;
	}

	public void setProductId(Long productId) {
		this.productId = productId;
	}

	public String getProductName() {
		return productName;
	}

	public void setProductName(String productName) {
		this.productName = productName;
	}

	public double getPrice() {
		return price;
	}

	public void setPrice(double price) {
		this.price = price;
	}

	public String getImageUrl() {
		return ImageUrl;
	}

	public void setImageUrl(String imageUrl) {
		ImageUrl = imageUrl;
	}


	public String getBrand() {
		return brand;
	}


	public void setBrand(String brand) {
		this.brand = brand;
	}


	public String getLink() {
		return link;
	}


	public void setLink(String link) {
		this.link = link;
	}


	public String getWebsite() {
		return website;
	}


	public void setWebsite(String website) {
		this.website = website;
	}
	


}

File: Graphql-Nodejs/scripts/domain/Product.js
module.exports = class Product {
  constructor(product_id, product_name, price, imageurl, brand, link) {
    this.product_id = product_id;
    this.product_name = product_name;
    this.price = price;
    this.imageurl = imageurl;
	this.brand = brand;
	this.link = link;
  }
  
  retailers() {
	var data = new global.RetailerDao().getRetailerByProduct(this.product_id);
	return data;  
  }
};